test/api/fetchResponse.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { fetchResponse } from '@/src/api';
+import { Response } from 'cross-fetch';
```

Comment:
`Response` doesn't exist in the node environment used in GitHub actions, so even though the test passes without this locally it fails in CI.

`cross-fetch` is acting a pre-built mock for the behavior of the browser `Response` object.

As far as the `fetchResponse`/`fetchJsonResponse` implementations go, the `Response` type just needs an `ok` property - nothing else is accessed/checked.

Reply:
Just bumped the node version, which was 16, to 18 to give support to fetch/Response